packages/components/radio/src/radio-group.ts

```diff
@@ -56,6 +57,12 @@ export const radioGroupProps = buildProps({
     type: Boolean,
     default: true,
   },
+  options: {
+    type: definePropType<Record<string, any>[]>(Array),
+  },
+  props: {
+    type: definePropType<SelectOptionProps>(Object),
+  },
```

Comment:
I would recommend this to be renamed as `optionProps` or `radioProps` to distinguish from already existing `props` here.

Reply:
I am also wondering how do you define the attributes for individual `radio` here? It seems like you can only define the `key` for `label/value/disabled`, what happens if I want to define attributes like `data-testid` against each `el-radio` here?